site/_data/fuguShowcase.js

```diff
@@ -21,24 +21,28 @@
 const path = require('path');
 const fs = require('fs');
 
-module.exports = () => {
+module.exports = (async () => {
   const fuguShowcaseFile = path.join(
     __dirname,
     '../../external/data/fugu-showcase.json'
   );
   const fuguItems = JSON.parse(fs.readFileSync(fuguShowcaseFile, 'utf-8'));
 
   const availableAPIs = new Set();
-  fuguItems.forEach(item => {
-    item.usedAPIs.forEach(api => {
-      availableAPIs.add(api.name);
-    });
-  });
-
+  await Promise.all(
```

Comment:
I think this does not need to be async anymore?

Reply:
Oh, right. That's a left-over. Cleaning it up.